flutter_local_notifications/example/lib/main.dart

```diff
@@ -191,6 +191,11 @@ Future<void> main() async {
   await flutterLocalNotificationsPlugin.initialize(
     initializationSettings,
     onSelectNotification: (String? payload) async {
+      await flutterLocalNotificationsPlugin
```

Comment:
Could you revert this change? Whilst I get this was originally done, it has a few issues
- it's not related to this PR
- results in an unnecessary call every time a notification is tapped on
- leads to inconsistent behaviour with when a notification is tapped on whilst app is running and when it's tapped but app wasn't running